content/acra/security-controls/searchable-encryption/_index.md

```diff
@@ -26,6 +26,10 @@ will work, but not
 SELECT ... FROM ... WHERE encrypted_column LIKE "prefix%"
 ```
 
+{{< hint info >}}
+AcraServer supports `SELECT/UPDATE/INSERT/DELETE` SQL statements for searchable encryption queries.
+{{< /hint >}}
+
```

Comment:
can we add paragraph about what Acra does under the hood with values and SQL queries? it is useful for us to leave notes about differences between MySQL and PostgreSQL, how we use substr and type casts.

Reply:
Decided to add this paragraph in the next PR related to the latest Acra changes.